ddtrace/contrib/botocore/services/bedrock.py

```diff
@@ -0,0 +1,297 @@
+import json
+import sys
+from typing import Any
+from typing import Dict
+from typing import List
+
+from ddtrace import Span
+from ddtrace.internal.llmobs.integrations import BedrockIntegration
+from ddtrace.vendor import wrapt
+
+from ....internal.schema import schematize_service_name
+
+
+_AI21 = "ai21"
+_AMAZON = "amazon"
+_ANTHROPIC = "anthropic"
+_COHERE = "cohere"
+_META = "meta"
+_STABILITY = "stability"
+
+
+class TracedBotocoreStreamingBody(wrapt.ObjectProxy):
+    #  Currently the corresponding span finishes only if
+    #  1) the user fully consumes the stream body
+    #  2) error during reading
+
+    def __init__(self, wrapped, span, integration):
```

Comment:
nit: It would be nice if TracedBotocoreStreamingBody methods had doc strings and typing